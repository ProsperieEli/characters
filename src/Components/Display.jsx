import React from "react";

export default function Display({ catchphrase }) {
  return (
    <div>
      {catchphrase.map((catchphase) => (
        <p key={catchphase}>{catchphase}</p>
      ))}
    </div>
  );
}
