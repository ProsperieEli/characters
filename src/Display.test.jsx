import { screen, render } from "@testing-library/react";
import Display from "./Components/Display";

it("Should render our Display component", () => {
  const { container } = render(<Display catchphrase={["tweet!"]} />);
  expect(container).toMatchSnapshot();
});
