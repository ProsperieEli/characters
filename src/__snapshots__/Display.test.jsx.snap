// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render our Display component 1`] = `
<div>
  <div>
    <p>
      tweet!
    </p>
  </div>
</div>
`;
